Utils/pom.xml

```diff
@@ -45,7 +45,8 @@
         <azure.identity.version>1.2.1</azure.identity.version>
         <azure.resourcemanager.version>2.1.0</azure.resourcemanager.version>
         <azure.toolkit-auth.version>${project.version}</azure.toolkit-auth.version>
-        <azure.version>1.38.0</azure.version>
+        <azure.tools-common.version>0.10.2</azure.tools-common.version>
```

Comment:
We have removed the dependency for tools-common, why add it back here?

Reply:
removed